### ISSUE TITLE
No error for duplicate type if one is exported

### ISSUE THREAD

**Issue by OliverJAsh** [microsoft/TypeScript#32216]
**TypeScript Version:**  3.5.2

**Search Terms:** exported export duplicate type

**Code**

```ts
export type MyObject = {
    foo: string;
};

// Expected: error due to duplicate
// Actual: no error
type MyObject = {
    bar: string;
};
```

**Comment by falsandtru** [comment]
Duplicate of #21913

**Comment by Andarist** [comment]
This was fixed in 3.8. The issue can be closed.
